scripts/archinstall/arch/archinstall.py

```diff
@@ -145,13 +155,19 @@ def install_bootloader(self, name, device, **kwargs):
 
     def install_grub(self, device, target='i386-pc', **kwargs):
         with ArchChroot(self.mnt):
+            # community/os-prober could be useful, the auto detection of windows
+            # won't work without it
             self.pkg_install(['grub'])
             if not os.path.exists('/boot/grub'):
                 os.mkdir('/boot/grub')
             self.run(['grub-mkconfig', '-o', '/boot/grub/grub.cfg'])
             self.run(['grub-install', '--target', target, device])
 
     def install_refind(self, device, efi_path='/EFI/refind/refind_x64.efi', **kwargs):
+        # @Chr0nos the esp can be mounted as /boot or /efi with current standard according to 
```

Comment:
that not what says the documentation: https://wiki.archlinux.fr/ESP
have you any more info about that ?

Reply:
The english doc is generally more up to date :
https://wiki.archlinux.org/index.php/EFI_system_partition#Mount_the_partition

Reply:
i'm not a big fan of a new / directory, but if it's the new standard, i guess i cant fight against it...